utils/woptipng.py

```diff
@@ -207,10 +286,13 @@ def optimize_image(image):
         summary_string = "optimized  {image}  from {size_initial} to {size_after}, {size_delta}b, {perc_delta}%"
         status = ProcessingStatus.OPTIMIZED
 
-    # If the file didn't shrink sufficiently, write back the original version
-    if status != ProcessingStatus.OPTIMIZED:
-        with open(image, 'wb') as f:
-            f.write(initial_file_content)
+    # If the file shrank sufficiently, write back the optimized version
+    if status == ProcessingStatus.OPTIMIZED:
+        shutil.copy(backup_image, image)
```

Comment:
IIUC, `backup_image` is the one being changed by being overwritten in `verify_images`, and `image` is the original, which IIUC hasn't been changed until this point.

Something either here or in `verify_images` should be changed so that the logic is consistent.